cartography/cli.py

```diff
@@ -311,6 +311,14 @@ def _build_parser(self):
                 'The port of your statsd server. Only used if --statsd-enabled is on. Default = UDP 8125.'
             ),
         )
+        parser.add_argument(
+            '--k8-kubeconfig',
```

Comment:
should be k8s (k8 == kubernet, k8s == kubernetes)

Reply:
Updated in later commit.